examples/island_cluster_anywhere_in_gcp_design/vars.tf

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "region" {
```

Comment:
For examples we should hard code most of these (rather than using tfvars) and only take project-id as input

Reply:
For this specific example, it helps to see all the network configurations in a single file.

Reply:
@nvnmandadhi in that case could you add descriptions to all vars too?

Reply:
done!